Dockerfile

```diff
@@ -32,35 +32,28 @@ ARG USER="dagu"
 ARG USER_UID=1000
 ARG USER_GID=$USER_UID
 
-# Create user and set permissions
-RUN apk update; \
-    apk add --no-cache sudo tzdata; \
-    addgroup -g ${USER_GID} ${USER}; \
-    adduser ${USER} -h /home/${USER} -u ${USER_UID} -G ${USER} -D -s /bin/ash; \
-    echo ${USER} ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/${USER}; \
-    chmod 0440 /etc/sudoers.d/${USER}; \
-    mkdir -p .config/dagu/dags; \
-    chown -R ${USER}:${USER} /home/${USER};
-
 COPY --from=go-builder /app/bin/dagu /usr/local/bin/
+COPY ./entrypoint.sh /entrypoint.sh
+
+# Create user and set permissions
+RUN apk update && \
+  apk add --no-cache su-exec shadow tzdata && \
+  addgroup -g ${USER_GID} ${USER} && \
+  adduser ${USER} -h /config -u ${USER_UID} -G ${USER} -D -s /bin/ash && \
+  chown -R ${USER}:${USER} /config && \
+  chmod +x /entrypoint.sh;
 
-USER ${USER}
-WORKDIR /home/${USER}
+WORKDIR /config
 
-# Add the hello_world.yaml file
-COPY --chown=${USER}:${USER} <<EOF .config/dagu/dags/hello_world.yaml
-schedule: "* * * * *"
-steps:
-  - name: hello world
-    command: sh
-    script: |
-      echo "Hello, world!"
-EOF
```

Comment:
This could never work as when a user binds over `/home/dagu/.config/dagu` it will mount over this file at this location. 

If there is a desire to have an automatically populated hello-world dag, it would have to happen during runtime on the first boot.